versions-model/src/site/markdown/xsd/index.md

```diff
@@ -11,3 +11,9 @@ Versions - XML schemas for ruleSet
 - [rule-2.1.0.xsd](rule-2.1.0.xsd)
 - [rule-2.0.0.xsd](rule-2.0.0.xsd)
 
+Change recorder updates
+=======================
+
+- [updates-3.0.xsd](updates-3.0.xsd)
+- [updates-2.0.xsd](updates-2.0.xsd)
```

Comment:
do we need both version ... does 2.0 will be deprecated?

Reply:
Yes, I think updates-2.0.xsd should be deprecated. We currently can generate both and updates-2.0.xsd is still default.

Reply:
Updated -- didn't squash.